enterprise_subsidy/apps/api_client/enterprise_catalog.py

```diff
@@ -0,0 +1,81 @@
+"""
+Enterprise Catalog api client for the subsidy service.
+"""
+import logging
+from urllib.parse import urljoin
+
+import requests
+from django.conf import settings
+
+from enterprise_subsidy.apps.api_client.base_oauth import BaseOAuthClient
+
+logger = logging.getLogger(__name__)
+
+
+class EnterpriseCatalogApiClient(BaseOAuthClient):
+    """
+    API client for calls to the enterprise service.
+    """
+    api_base_url = urljoin(settings.ENTERPRISE_CATALOG_URL, 'api/v1/')
```

Comment:
we'll need to set this config in edx-internal too right?

Reply:
https://github.com/edx/edx-internal/pull/7974/files done